terraform/alerting/slos.tf

```diff
@@ -14,59 +14,75 @@
 
 locals {
   default_per_service_slo = {
-    availability_goal = 0.995
-    latency = {
-      goal      = 0.9
-      threshold = 60000
-    }
     enable_alert = false
+    availability_goal = 0.995
+    latency_goal = 0.9
+    latency_threshold = 60000
   }
-  default_slo_thresholds = {
-    adminapi     = merge(local.default_per_service_slo, { enable_alert = true })
-    apiserver    = merge(local.default_per_service_slo, { enable_alert = true })
+  service_configs = {
+    adminapi     = merge(local.default_per_service_slo,
+                        { enable_alert = true ,
+                          latency_goal = 0.99 ,
```

Comment:
Make this latency_goal part of the default?

Reply:
Done, thank you.